### ISSUE TITLE
Broken Text Conversion

### ISSUE THREAD

**Issue by Vadoola** [Vadoola/dxf2elmt#18]
The last thing I think I need to fix with the blocks is the text in blocks didn't convert...but then I noticed I seem to have broken text conversion entirely....

In my test block with text the text just doesn't convert at all....but in another file, no blocks but lots of text the text elements appear, but all show with text "_"????. I grabbed an older copy before some of my more dramatic changes and the text conversion absolutely was working.

**Comment by Vadoola** [comment]
Ah so this look like a bug from antonioaja's code that my changes bubbled to the surface. In the original code dynamic_text support was added but disabled by default. It looks like it never fully worked correctly.  He incorrectly assumed the text value in dynamic text was just an text= attribute like on a normal text element but it isn't, it's a child. As part of my changes, I switched to dynamic text by default, just assuming it had worked before.
